src/ShipSnapshotProvider/ShipSnapshotProvider.tsx

```diff
@@ -109,8 +110,11 @@ export interface ShipSnapshotProps {
   children: React.ReactNode;
   /** A ship fit in ESI representation. */
   fit: EsiFit;
-  /** A list of skills to apply to the fit: {skill_id: skill_level}. */
-  skills: Record<string, number>;
+  /**
+   * A list of skills to apply to the fit: {skill_id: skill_level}.
+   * If undefined, falls back to skills from EsiProvider
+   **/
+  skills?: Record<string, number>;
```

Comment:
Why do we need two ways? Can't we just remove this way now?

In general I really tried to avoid storing data twice, or over two providers. It is just more room for making mistakes and things getting desyncd.